libs/profile/client/src/lib/profile.client.ts

```diff
@@ -86,7 +85,10 @@ export class ProfileClient {
         'delete'
       );
     } catch (err) {
-      this.log.error('profile.deleteCache.failed', uid, err);
+      this.log.error('profileClient.deleteCache.failed', {
+        uid,
+        error: err.toString(),
+      });
```

Comment:
I thought for `log.error`, the first argument needs to be an `Error` object?

Reply:
It does, yes. I'll go through the PR this morning to see if I can catch anything else. It's changed a bit over the time its been worked on haha

Reply:
Ah yes fair enough. Sorry for jumping in a bit early.

Reply:
This still looks open?